src/pages/login/index.page.tsx

```diff
@@ -1,5 +1,99 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import { SubmitErrorHandler, SubmitHandler, useForm } from 'react-hook-form';
+import { z } from 'zod';
+import Input from '@/components/Inputs/Input';
+import DatePicker from '@/components/Inputs/DatePicker';
+
+type IFormValues = {
+  email: string;
+  password: string;
+  passwordConfirm: string;
+  ReactDatepicker: Date;
+};
+
+const schema = z
+  .object({
+    email: z
+      .string()
+      .min(1, { message: '이메일을 입력해 주세요.' })
+      .email({ message: '이메일 형식으로 작성해 주세요.' }),
+    password: z
+      .string()
+      .min(1, { message: '비밀번호를 입력해 주세요.' })
+      .min(8, { message: '8자 이상 입력해주세요.' }),
+    passwordConfirm: z.string().min(1, '비밀번호를 입력해 주세요.'),
+    ReactDatepicker: z.date(),
+  })
+  .refine((data) => data.password === data.passwordConfirm, {
+    path: ['passwordConfirm'],
+    message: '비밀번호가 일치하지 않습니다.',
+  });
+
 function Login() {
-  return <div>Login</div>;
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    control,
+  } = useForm<IFormValues>({
+    resolver: zodResolver(schema),
+    mode: 'onChange',
+    defaultValues: {
+      email: '',
+      password: '',
+      passwordConfirm: '',
+      ReactDatepicker: new Date(),
+    },
+  });
+  const onValid: SubmitHandler<IFormValues> = (data) => {
+    console.log(data);
+  };
+  const onInvalid: SubmitErrorHandler<IFormValues> = (data) => {
+    console.log(data);
+  };
+  return (
+    <form onSubmit={handleSubmit(onValid, onInvalid)}>
+      <div>
+        <div>
+          <Input
+            label="이메일"
+            required
+            id="email"
+            type="email"
+            register={register('email')}
+            placeholder="이메일을 입력해주세요."
+            errors={errors}
+          />
+        </div>
+        <div>
+          <Input
+            id="password"
+            icon="search"
```

Comment:
```suggestion
            icon="search"
```
아래 PasswordConfirm에 눈 모양 아이콘은 Icon으로 넘겨줄 수 없나요??

Reply:
넵 맞습니다 둘 다 아이콘인데 방식이 다른 이유가 있을까요??

Reply:
> 넵 맞습니다 둘 다 아이콘인데 방식이 다른 이유가 있을까요??

저기서 넘겨 받는 Icon은 여기 앞에 붙는 아이콘이라서 사용처에서 주입해서 커스텀 할 수 있게 변경했는데
![스크린샷 2024-04-18 171544](https://github.com/19-Takify/19-taskify/assets/98067115/78e49d48-556d-4d1a-aa39-f503b86a4dbc)

비밀번호 보이기/가리기 이미지는 `password`일 때만 동작하게 만들어서 해당 이미지도 외부에서 주입하게 만들면 Input 내부랑 사용법이 너무 복잡해 질 거 같다는 판단을 했습니다!